### ISSUE TITLE
Uncheck manual override checkbox on cluster update

### ISSUE THREAD

**Issue by CydeWeys** [gbrodman/order-tracking#131]
I just had two orders merged together by common order email, one of their manual override checkboxes was checked, and this state persisted on into the merged row (which is clearly wrong, especially because there was now a new tracking # that just shipped that isn't yet reimbursed).

I think what we want is that, if any cluster is changed for any reason, e.g. more reimbursement, another tracking added, or a merge, then the manual override status should go away, thus potentially flagging the row for human review again.

**Comment by CydeWeys** [comment]
I think this should be good.  Will close out after I've been running the new code for awhile and don't notice a recurrence of the issue.

**Comment by CydeWeys** [comment]
This is still occurring unfortunately.

**Comment by CydeWeys** [comment]
It's happening on below-cost orders split into sub-orders at time of ordering.  One sub-order ships and is received, and I override it, and then the other sub-order ships, gets merged with the previous sub-order because of the common email, and the merged cluster remains overridden even though it's now missing potentially thousands of $$$ in non-checked-in items.

**Comment by CydeWeys** [comment]
Hah, I even got these log messages:

```
Merged orders {'111-222222-33333333'} and {'111-222222-444444444'} by common email IDs {'55555'}
Newly merged cluster {'111-222222-33333333', '111-222222-444444444'} manual override unset.
```

So it looks like my code is trying to do the right thing ... but yet it's not happening?!
